Source/WebCore/Modules/fetch/FetchBodyConsumer.h

```diff
@@ -30,7 +30,7 @@
 
 #include "FetchBodySource.h"
 #include "FormDataConsumer.h"
-#include "JSDOMPromiseDeferred.h"
```

Comment:
I wonder if we could `check-webkit-style` about `JSDOMPromiseDeferred.h` ever appearing in a header file?

Reply:
We could. There are places where I left the full header in; mostly for entirely inlined header files and the like. But people doing that could ignore the style check.